### PR TITLE
F/ci

### PR DESCRIPTION
**Pull request by mbartling**
Add CI :)

### CODE REVIEW

**Comment by mbartling** [comment]
@dboyliao bump :)

**Comment by mbartling** [comment]
bump

File: README.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ You can go to this [repo](https://github.com/pypa/pipenv) for detail information
                                                            |
                  +-------------------------------+         |
                  |       graph transformer       |         |
-                 | (legalization & optimization) | <------‘ 
+                 | (legalization & optimization) | <------/
```
Comment (dboyliao) [inline README.md]: I prefer original `‘` XDDD

File: Dockerfile-ci [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    git \
+    python3 \
+    python3-pip \
+    gcc \
+    wget \
+    lcov \
+    g++ 
+RUN pip3 install --upgrade pip setuptools wheel
+COPY requirements.txt .
+RUN pip3 install -r requirements.txt
```
Comment (dboyliao) [inline Dockerfile-ci]: I thinks as for CI, it's better to install the package with `python3 setup.py install`.

Comment (dboyliao) [reply Dockerfile-ci]: So we can check if it will works on `PyPI`.

File: .circleci/config.yml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  build:
+    docker:
+      - image: mbartling/utensor_cgen_ci
+    steps:
+      - checkout
+      - run: mkdir -p /tmp/coverage && mkdir -p /tmp/test_results
+      - run: pip3 install -e .[dev]
+      - run: 
+          name: Test
+          command: pytest -vv
```
Comment (dboyliao) [inline .circleci/config.yml]: `pytest -vv tests`

File: README.md (hunk 1 above)
Comment (mbartling) [reply README.md]: The original `'` is a non standard unicode char and was breaking CI for a very silly reason. That's the only reason I changed it :)

File: .circleci/config.yml (hunk 3 above)
Comment (mbartling) [reply .circleci/config.yml]: Fixed

File: Dockerfile-ci (hunk 2 above)
Comment (mbartling) [reply Dockerfile-ci]: Fixed

**Comment by mbartling** [comment]
@dboyliao I fixed the two things you proposed, but cannot change the first one else CI explodes for a dumb reason

**Comment by dboyliao** [comment]
@mbartling 
Sorry for overlooking this PR.
I'm good with `/` now
Go ahead and fix the conflict and I'll merge it.

**Pull request merged**